metricflow/plan_conversion/dataflow_to_sql.py

```diff
@@ -73,7 +73,7 @@
     ColumnEqualityDescription,
     SqlQueryPlanJoinBuilder,
 )
-from metricflow.plan_conversion.time_spine import TimeSpineSource
+from metricflow.plan_conversion.time_spine import TIME_SPINE_DATA_SET_DESCRIPTION, TimeSpineSource
```

Comment:
Now I'm reading this file! Have to stop for the night though, will pick up tomorrow.